_posts/2020-03-04-become-better-estimators.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: Using Actual Story Points to Become Better Estimators
+date: 2020-03-10 14:30:00
+categories: [agile, scrum]
+author: abizzinotto
+
+---
+
+At OmbuLabs, we strongly believe in continuous improvement. 
+
+As a Scrum team, before starting a sprint we estimate the complexity of the stories using the Fibonacci scale. But how do we become better at estimating stories? How do we know we're evolving as a team?
+
+In this article I outline the process we are following to become better estimators.
+
+<!--More-->
+
+# The Projects
+
+At OmbuLabs we work on a number of different, challenging projects. We do, however, also work on very similar projects, namely, Rails upgrade projects. Therefore, we identified the critical path we need to follow on every Rails upgrade from version A to version B and used it to create a template we can follow whenever we work on a project of this nature.
+
+# The Blind Estimation
+
+Once we have a project in our hands, we turn the steps in our Rails upgrade template into stories. Since we are a remote team, we developed an internal tool to facilitate the blind estimation process. Once the stories are created, two developers add their estimates for each story. 
+
+To preserve the blind nature of the estimation, the developer estimating can only see the page to input their estimates. They cannot access anyone else's estimates.
+
+Once both developers have inputted their estimates, the Scrum master can go in and generate a report detailing the estimates from each developer and the average estimate for each story.
+
+# Handling Discrepancy
+
+There will always be some discrepancy between estimates. After all, there were two different people estimating. However, sometimes the discrepancy is too big. How do we handle that?
+
+As a remote team, the estimation process isn't as straight forward as planning poker done in a meeting. So once the Scrum master reviews the estimates, if there are any stories with estimates that are too far apart, we have a "reconciliation call". During this call, the developers get a chance to talk about the story / stories in question and reach closer estimates.
+
```

Comment:
@ABizzinotto I really like the idea of the reconciliation call 👍 

Maybe you can expand a bit on that and explain that developers explain why they think is so complicated/simple. What's under the tip of "the iceberg"? 😄

Reply:
Done :) Could you review it @etagwerker ?